openmetadata-spec/src/main/resources/json/schema/email/smtpSettings.json

```diff
@@ -25,6 +25,10 @@
       "description": "Openmetadata Server Endpoint",
       "type": "string"
     },
+    "senderMail": {
+      "description": "Mail of the sender",
+      "type": "string"
```

Comment:
We should use strong types for email address  type instead of string (this is something we should do with all the configuration json where we are currently using string as type.

See email type from basic.json:
```
    "email": {
      "$comment" : "@om-field-type",
      "description": "Email address of a user or other entities.",
      "type": "string",
      "format": "email",
      "pattern": "^\\S+@\\S+\\.\\S+$",
      "minLength": 6,
      "maxLength": 127
    },
```